src/types/configuration/github-comment-config.ts

```diff
@@ -0,0 +1,9 @@
+import { Static, Type } from "@sinclair/typebox";
+
+export const githubCommentConfigurationType = Type.Object({
+  enabled: Type.Boolean({ default: true }),
+  post: Type.Boolean({ default: true }),
+  debug: Type.Boolean({ default: false }),
```

Comment:
It would be good also to document what each of those configuration types mean, can be above the code.

Reply:
That's a good idea, I will add comments.